demo/odaslive/parameters.c

```diff
@@ -204,6 +236,18 @@
 
                 cfg->interface = interface_construct_soundcard_by_name(tmpStr2);
 
+                free((void *) tmpStr2);
```

Comment:
The callee was keeping a pointer to the string.
Fixed, now it takes a copy like all other `interface_construct_*`, so we free the string